sunpy/net/attr.py

```diff
@@ -92,24 +94,37 @@ def __dir__(self):
         """
         return super().__dir__() + self._attr_registry[self].name
 
+    def __repr__(self):
+        """
+        Returns the normal repr plus the pretty attr __str__.
+        """
+        return object.__repr__(self) + "\n" + str(self)
+
     def __str__(self):
         """
         This enables the "pretty" printing of Attrs.
         """
-        name = ['Attribute Name'] + self._attr_registry[self].name
-        name_long = ['Full Name'] + self._attr_registry[self].name_long
-        desc = ['Description'] + self._attr_registry[self].desc
-        pad_name = max(len(elm) for elm in name)
-        pad_long = max(len(elm) for elm in name_long)
-        pad_desc = max(len(elm) for elm in desc)
-        fmt = f"%-{pad_name}s | %-{pad_long}s | %-{pad_desc}s"
-        lines = [fmt % elm for elm in zip(name, name_long, desc)]
-        lines.insert(1, '-' * (pad_name + 1) + '+' + '-' * (pad_long + 2) + '+' + '-' * (pad_desc + 1))
-        return "\n".join(lines)
+        class_name = self.__name__
+        attrs = self._attr_registry[self]
+        names = [x for _, x in sorted(zip(attrs.name, attrs.name), key=lambda pair: pair[0])]
+        clients = [x for _, x in sorted(zip(attrs.name, attrs.client), key=lambda pair: pair[0])]
+        names_long = [x for _, x in sorted(
+            zip(attrs.name, attrs.name_long), key=lambda pair: pair[0])]
+        descs = [x for _, x in sorted(zip(attrs.name, attrs.name_long), key=lambda pair: pair[0])]
```

Comment:
We probably want to truncate desc to a length corresponding to the console width (as opposed to using astropy's default behaviour which drops columns).

Reply:
How would I do that?